patches/net/minecraft/world/effect/MobEffect.java.patch

```diff
@@ -9,6 +9,15 @@
      public static final Codec<Holder<MobEffect>> CODEC = BuiltInRegistries.MOB_EFFECT.holderByNameCodec();
      public static final StreamCodec<RegistryFriendlyByteBuf, Holder<MobEffect>> STREAM_CODEC = ByteBufCodecs.holderRegistry(Registries.MOB_EFFECT);
      private static final int AMBIENT_ALPHA = Mth.floor(38.25F);
+@@ -48,6 +_,8 @@
+     private int blendDurationTicks;
+     private Optional<SoundEvent> soundOnAdded = Optional.empty();
+     private FeatureFlagSet requiredFeatures = FeatureFlags.VANILLA_SET;
++    private it.unimi.dsi.fastutil.objects.ReferenceSet<net.neoforged.neoforge.flag.Flag> requiredFlags = new it.unimi.dsi.fastutil.objects.ReferenceOpenHashSet<>();
++    private java.util.Set<net.neoforged.neoforge.flag.Flag> requiredFlagsView = it.unimi.dsi.fastutil.objects.ReferenceSets.unmodifiable(requiredFlags);
```

Comment:
Is there a measurable performance reason to reuse the unmodifiable view versus just creating it on-demand?